src/networks/types/EWT.js

```diff
@@ -0,0 +1,22 @@
+import tokens from '@/_generated/tokens/tokens-ewt.json';
+import contracts from '@/_generated/contracts/contract-abi-ewt.json';
+import clo from '@/assets/images/networks/ewt.svg';
```

Comment:
import name isn't being used. Change it to `ewt`

Reply:
Updated